controllers/multicluster/remotecluster/controller.go

```diff
@@ -189,7 +189,7 @@ func (r *Reconciler) ensureHelmRelease(ctx context.Context, obj *v1alpha1.Remote
 	_, err = helmClient.GetRelease(obj.Spec.Namespace, chaosMeshReleaseName)
 	if err != nil {
 		if errors.Is(err, driver.ErrReleaseNotFound) {
-			chart, err := helm.FetchChaosMeshChart(ctx, chaosMeshReleaseVersion)
+			chart, err := helm.FetchChaosMeshChart(ctx, obj.Spec.Version, config.ControllerCfg.LocalHelmChartPath)
```

Comment:
The logic about fetching  `Status.CurrentVersion` needs updates.

Fields in `Status` means the **observed status** by the controller, it should NOT be setup by simply copying the value from `Spec.Version`.

What about making a refactor on `helm.FetchChaosMeshChart`, derive 2 method like:

- helm.FetchChaosMeshChartFromRepository(ctx context.Context, version string) (chart, version ,err)
- helm.FetchChaosMeshChartFromLocal(ctx context.Context, helmChartPath string) (chart, version ,err)

then setup the `Status.CurrentVersion` with the returned `version` value.